doc/radosgw/lua-scripting.rst

```diff
@@ -310,6 +310,9 @@ while execution and this data could be read and used later in any other executio
 
```

Comment:
Please add:
"The maximum number of entries in the table is 100,000. Each entry could has a key and values strings of no more than 1KB (together)"